framework/test/jest/unit/specs/modules/chain/blocks/blocks.spec.js

```diff
@@ -1703,12 +1704,113 @@ describe('blocks', () => {
 		});
 	});
 
-	describe('loadBlocksDataWs', () => {
-		it.todo('a');
-	});
+	describe('loadBlocksFromLastBlockId', () => {
+		describe('when called without lastBlockId', () => {
+			it('should reject with error', async () => {
```

Comment:
There is no test for checking when called without storage object.

Reply:
the API is called without storage. so not possible to have that case 😉